examples/employee-app/resources/DbCreds.properties

```diff
@@ -0,0 +1,5 @@
+# Oracle DB Credentials
+user=<user-db>

```

Comment:
This file has some windows line ending. Make sure you configure your git client to convert line ending to unix style automatically.

Reply:
We removed this file.